lib/src/impl_ffi/impl_ffi.random.dart

```diff
@@ -30,4 +30,28 @@ final class _RandomImpl implements RandomImpl {
       dest.setAll(0, out.asTypedList(dest.length));
     });
   }
+
+  @override
+  String randomUUID() {
+    return _Scope.sync((scope) {
+      final out = scope<ffi.Uint8>(16);
+      _checkOp(ssl.RAND_bytes(out, 16) == 1);
+
+      var bytes = out.asTypedList(16);
```

Comment:
Maybe, we could just use `fillRandomBytes`, then we have less FFI stuff going on.